nextjs-app/types/schedule.ts

```diff
@@ -0,0 +1,44 @@
+import { ButtonColor, ButtonVariant } from "@/components/common/Button";
+
+export enum IScheduleType {
```

Comment:
```suggestion
export enum ScheduleType {
```

For enum, `ScheduleType` should be a better name